internal/persistence/fake/mock_client.go

```diff
@@ -0,0 +1,51 @@
+package fake
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	"github.com/artefactual-sdps/enduro/internal/package_"
+	"github.com/artefactual-sdps/enduro/internal/persistence"
+)
+
+var (
+	CreatedAt = time.Unix(1694213364, 0) // 2023-09-08T22:49:24+00:00
+	StartedAt = time.Unix(1694213435, 0) // 2023-09-08T22:50:35+00:00
+)
+
+type mockClient struct{}
```

Comment:
I would use mockgen for consistency but also it reduces the maintenance overhead significantly.